src/main/java/com/microsoft/azure/spring/data/cosmosdb/config/DocumentDBConfig.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.data.cosmosdb.config;
+
+import com.microsoft.azure.documentdb.ConnectionPolicy;
+import com.microsoft.azure.documentdb.ConsistencyLevel;
+import lombok.Getter;
+
```

Comment:
you can put `@Getter` here for all fields.

Reply:
As well as `@Builder`

Reply:
Updated with lombok annotation.